riscv-asm.md

```diff
@@ -183,6 +183,26 @@ Directive    | Arguments                      | Description
 .balign      | b,[pad_val=0]                  | byte align
 .zero        | integer                        | zero bytes
 
+### <a name=.option></a> `.option`
+
+#### `rvc`/`norvc`
+
+Enable/disable C-extension for the following code region.
```

Comment:
"C-extension" -> "The C extension"

I don't know if the word "region" in "the following code region" kind of implies that this has limited scope, like an `#if ... #endif` region. I'll let the native speakers nitpick that :)

Reply:
@aswaterman did you having some suggestion on the wording about `region`?